config/samples/kustomization.yaml

```diff
@@ -1,4 +1,5 @@
 ## Append samples you want in your CSV to this file as resources ##
 resources:
 - octavia_v1beta1_octaviaapi.yaml
```

Comment:
Do we still need octavia_v1beta1_octaviaapi.yaml as a separate yaml file or should be better merge it into one?

Reply:
Good point. Let's do away with it.